Svc/SeqDispatcher/SeqDispatcher.cpp

```diff
@@ -0,0 +1,210 @@
+// ======================================================================
+// \title  SeqDispatcher.cpp
+// \author zimri.leisher
+// \brief  cpp file for SeqDispatcher component implementation class
+// ======================================================================
+
+#include <Svc/SeqDispatcher/SeqDispatcher.hpp>
+
+namespace Svc {
+
+// ----------------------------------------------------------------------
+// Construction, initialization, and destruction
+// ----------------------------------------------------------------------
+
+SeqDispatcher ::SeqDispatcher(const char* const compName)
+    : SeqDispatcherComponentBase(compName) {}
+
+SeqDispatcher ::~SeqDispatcher() {}
+
+void SeqDispatcher::init(NATIVE_INT_TYPE queueDepth,
```

Comment:
As a note here, we've been removing the `init` function altogether.  It isn't needed, as it calls the parent class function.

Custom setup is best done in a dedicated `configure` or `setup` function anyway, as FPP expects `init` to have only default arguments.

Reply:
Heya, I added this to the `init` function based on @Joshua-Anderson's suggestion:
"Instead of checking if the port is connected on use, it might make sense to check if the sequence engine is connected on initialization and asserting if not - letting users know if the topology is misconfigured as quickly as possible."
I can move this to a different function but I think it would be weird to require the user to do extra work and call it if all it does is check if the ports are connected. Is there another place I could put this checking code so that it will run at bootup?